gammapy/estimators/map/ts.py

```diff
@@ -236,10 +236,9 @@ def estimate_kernel(self, dataset):
         model = self.model.copy()
         model.spatial_model.position = geom.center_skydir
 
-        # Creating exposure map with exposure at map center
+        # Creating exposure map with the mean non-null exposure
         exposure = Map.from_geom(geom, unit="cm2 s1")
```

Comment:
This is still unaddressed?

Reply:
Note that this is a pre-existing code, and possibly buggy indeed because what is passed is the `exposure.data`. 
A possible solution is to replace `"cm2s"` by `dataset.exposure.unit`

Reply:
But this contradicts a bit the initial comment of Quentin... What's happening if `data.exposure.unit` is not in `"cm2s"`?
This code allows to fix this case...
What is the consensus then?